product_docs/docs/pgd/4/rel_notes/pgd_4.3.5_rel_notes.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.3.5"
+navTitle: "Version 4.3.5"
+redirects:
+  - /pgd/latest/bdr/release_notes/bdr4.3.5_rel_notes/
+---
+
+Released: 14 May 2024
+
+EDB Postgres Distributed version 4.3.5 is a patch release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+!!! Note
+This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.10, 15.5 and later.
+!!!
+
+| Component | Version | Type        | Description                                                                                        | Addresses |
+|-----------|---------|-------------|----------------------------------------------------------------------------------------------------|-----------|
+| HARP      | 2.4.1   | Change      | Routine security library upgrades.                                                                 |           |
+| BDR       | 4.3.5   | Enhancement | Log completion of an extension upgrade.                                                            |           |
+| BDR       | 4.3.5   | Enhancement | Execute each autopartition task in its own transaction.                                            |           |
+| BDR       | 4.3.5   | Enhancement | Use `DETACH CONCURRENTLY` to drop partitions.                                                      | RT101407/35476  |
+| BDR       | 4.3.5   | Enhancement | Grant usage of `pg_file_settings` and `pg_show_all_file_settings()` to bdr_superuser.              |           |
+| BDR       | 4.3.5   | Enhancement | Stop the initial replication connection for `bdr_init_physical` and start it only when needed.     | RT102828/35305  |
+| BDR       | 4.3.5   | Bug Fix     | Improve handling of node group configuration parameter "check_constraints".                        | RT99956/31896   |
+| BDR       | 4.3.5   | Bug Fix     | Prevent potential segfault in `bdr.monitor_group_versions()`.                                      | RT102290/34051  |
```

Comment:
```suggestion
| BDR       | 4.3.5   | Bug Fix     | Fixed an issue to prevent potential segfault in `bdr.monitor_group_versions()`.                                      | RT102290/34051  |
```